workitem/workitem_repository.go

```diff
@@ -18,14 +18,19 @@ import (
 )
 
 const orderValue = 1000
-const above = "above"
-const below = "below"
+
+type DirectionType string
```

Comment:
I assume you mean why re-define it in the blackbox_test. This is actually the correct location :)